src/App.js

```diff
@@ -0,0 +1,28 @@
+import { Route, Routes } from 'react-router-dom';
+import BookForm from './components/BookForm';
+import BookList from './components/BookList';
+import Categories from './components/Categories';
+import './App.css';
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <div className="App">
+      <Navbar />
+      <Routes>
+        <Route
+          path="/"
+          element={(
+            <>
+              <BookList />
+              <BookForm />
+            </>
+        )}
+        />
```

Comment:
- [ ] Ouch! this is very close, but as part of the project requirements each route should render a page component (which is singular) so instead of routing the `/` path to multiple components I'd suggest you create a component called `<Books>` which will be the parent component for `<BookList />` and `<BookForm />`, so at the end you'll have something like the below 👇
```jsx
<Route path="/" element={<Books/>} />
```

Reply:
Addressed